vue-frontend/src/components/about/HowItAllStartedSection.vue

```diff
@@ -6,7 +6,10 @@
       How it all started?
     </h1>
     <div
-      class="tw-flex tw-flex-row tw-items-center tw-mx-auto tw-w-full 2xl:tw-w-[1535px] xll:tw-w-[1530px] 3xl:tw-w-[1595px] xl:tw-pt-20"
+      :class="
+        width > 1399 && width < 1536 ? '2xl:!tw-w-full' : '2xl:!tw-w-[1535px]'
```

Comment:
I think this is not a feasible solution, first, find the cause of why this happening.
If issue is with width, consider breakpoints for that, instead of manual wodth